src/HttpHeader.cc

```diff
@@ -1584,8 +1583,7 @@ HttpHeaderEntry::parse(const char *field_start, const char *field_end, const htt
      */
     for (const char *pos = field_start; pos < (field_start+name_len); ++pos) {
         if (!CharacterSet::TCHAR[*pos]) {
-            debugs(55, 2, "found header with invalid characters in " <<
-                   Raw("field-name", field_start, min(name_len,100)) << "...");
+            debugs(55, 2, "found invalid characters in " << Raw("field-name", field_start, name_len).atMost(100));
```

Comment:
Since we are changing this line, I would report the character position as well. Something like this may work (untested):


```suggestion
            debugs(55, 2, "found invalid character at position " << (pos-field_start) << " in " << Raw("field-name", field_start, name_len).atMost(100));
```

I do not insist on this improvement.